settings/sumfields.settings.php

```diff
@@ -125,4 +125,16 @@
     'description' => 'List of active fields that will be active when the cron job making the change completes.',
     'help_text' => '',
 	),
+  'process_type' => array(
+    'group_name' => 'Summary Fields',
```

Comment:
Oops - a process_type escaped the last set of changes - can this be changed to data_update_method?

Reply:
Good catch! :+1: